core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1421,7 +1421,8 @@ public boolean convertPartitionParquetToNative(long partitionTimestamp) {
         // path is now pointing to the parquet file
         // other is pointing to the new partition folder
         LOG.info().$("converting parquet partition to native [path=").$substr(pathRootSize, path).I$();
-        long parquetFd = TableUtils.openRO(ff, path.$(), LOG);
+        final long parquetFd = TableUtils.openRO(ff, path.$(), LOG);
+        final long readSize = ff.length(parquetFd);
```

Comment:
Probably not correct.

Reply:
The table writer here should have exclusive access to the partition.
What if we introduce a partition compactor job later?